proto/babylon/mint/v1/genesis.proto

```diff
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package babylon.mint.v1;
+
+option go_package = "github.com/babylonlabs-io/babylon/x/mint/types";
+
+// GenesisState defines the mint module's genesis state.
+message GenesisState {
+  reserved 1; // 1 was previously used for the `Minter` field.
```

Comment:
why not keep the minter? https://github.com/cosmos/cosmos-sdk/blob/b49f948b36bc991db5be431607b475633aed697e/proto/cosmos/mint/v1beta1/genesis.proto#L11

I think this is good information

```
  // minter is a space for holding current inflation information.
  Minter minter = 1 [(gogoproto.nullable) = false, (amino.dont_omitempty) = true];

```

and we could migrate the info during the upgrade

Reply:
I think this is because in this inflation protocol, the minter object does not depend on the minter object in the previous block, so the protocol does not need to remember the minter in genesis at all. This is different from Cosmos SDK's design where the new inflation depends on the inflation at the previous block.

Reply:
but what about the minted values during the previous version?